tmeister/static/src/toggles/delete/delete-modal-metrics.component.js

```diff
@@ -11,10 +20,24 @@ export default function DeleteFeatureModal (props) {
         {
           Object.keys(metrics).map((key) => {
             const countTimes = sumBy(metrics[key], i => i.hit_count)
-            const dayCount = metrics[key].length
+            const dayArray = metrics[key].map(obj => dayjs(obj.date))
+              .sort((a, b) => {
+                if (a.isBefore(b)) {
+                  return -1
+                } else {
+                  return 1
+                }
+              })
+            const dayCount = get(metrics, `[${key}].length`, 0)
+            const dateRange = dayArray.length > 1 ?
+              `${dayArray[0].calendar(null, calendarSettings)} - ${dayArray[dayArray.length -1].calendar(null, calendarSettings)}` :
+              dayArray[0].calendar(null, calendarSettings)
```

Comment:
I _do_ see `calendar usage here, but no plugin/extension added to `dayjs` to support it in this file

Reply:
Correct. Dayjs is immutable in objects but not in configuration so extending dayjs to support `calendar` in `toggle-meister.js` meant it supports it everywhere. 

It was trippy to me as well.